### PR TITLE
fix: filters - add blocks, fix names and layout

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue

Closes [#1197](https://vsf.atlassian.net/browse/SFUI2-1197)

# Scope of work

- changed names of filters, 
- added `Brand` and `Price` blocks, 
- fix widths on mobile and desktop

# Screenshots of visual changes

Screenshot from 2023-06-01 08-45-36

# Checklist

- [x] Self code-reviewed
- [x] Changes documented
- [x] Semantic HTML
- [x] SSR-friendly
- [x] Caching friendly
- [x] a11y for WCAG 2.0 AA
- [x] examples created
- [x] blocks created
- [x] cypress tests created

### CODE REVIEW

File: apps/preview/nuxt/pages/showcases/Filters/Brand.vue [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      :key="id"
+      tag="label"
+      size="sm"
+      :class="['px-1.5 bg-transparent hover:bg-transparent', { 'font-medium': isItemActive(value) }]"
+    >
+      <template #prefix>
+        <SfCheckbox v-model="selectedFilters" :disabled="counter === 0" :value="value" />
+      </template>
+      <p>
+        <span class="mr-2 text-sm">{{ label }}</span>
```
Comment (justyna-13) [inline apps/preview/nuxt/pages/showcases/Filters/Brand.vue:14]: Label for disabled one should be gray and they look like they aren't centred
Screenshot 2023-05-31 at 08 36 12

File: apps/preview/nuxt/pages/showcases/Filters/Price.vue [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    <fieldset id="radio-price">
+      <SfListItem
+        v-for="{ id, value, label, counter } in details"
+        :key="id"
+        tag="label"
+        size="sm"
+        class="px-1.5 bg-transparent hover:bg-transparent"
+      >
+        <template #prefix>
+          <SfRadio
```
Comment (justyna-13) [inline apps/preview/nuxt/pages/showcases/Filters/Price.vue:12]: I think it's not centred according to label position

**Review by justyna-13 — COMMENTED** [review]
Good job!

File: apps/preview/nuxt/pages/showcases/Filters/Brand.vue (hunk 1 above)
Comment (AdamPawlinski) [reply apps/preview/nuxt/pages/showcases/Filters/Brand.vue:15]: thx, fixed

**Review by justyna-13 — APPROVED** [review]

File: apps/preview/next/pages/showcases/Filters/Brand.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+const details = [
+  { id: 'b1', label: 'Conroy', value: 'conroy', counter: 10 },
+  { id: 'b2', label: 'Goyette', value: 'goyette', counter: 100 },
+  { id: 'b3', label: 'Ledner & Ward', value: 'lednerward', counter: 0 },
+  { id: 'b4', label: 'Pacocha', value: 'pacocha', counter: 3 },
+];
+
+export default function BrandFilter() {
+  const [opened, setOpened] = useState(true);
```
Comment (dkacper) [inline apps/preview/next/pages/showcases/Filters/Brand.tsx:16]: This is perfect use case for [`useDisclosure`](https://docs.storefrontui.io/v2/vue/hooks/useDisclosure.html).

File: apps/preview/next/pages/showcases/Filters/Price.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+const details = [
+  { id: 'pr1', label: 'Under $24.99', value: 'under', counter: 123 },
+  { id: 'pr2', label: '$25.00 - $49.99', value: '25-49', counter: 100 },
+  { id: 'pr3', label: '$50.00 - $99.99', value: '50-99', counter: 12 },
+  { id: 'pr4', label: '$100.00 - $199.99', value: '100-199', counter: 3 },
+  { id: 'pr5', label: '$200.00 and above', value: 'above', counter: 18 },
+];
+
+export default function PriceFilter() {
+  const [opened, setOpened] = useState(true);
```
Comment (dkacper) [inline apps/preview/next/pages/showcases/Filters/Price.tsx:17]: [`useDisclosure`](https://docs.storefrontui.io/v2/vue/hooks/useDisclosure.html)

**Review by dkacper — APPROVED** [review]
LGTM ⭐ 
Just left comments about minor refactor.

File: apps/preview/next/pages/showcases/Filters/Brand.tsx (hunk 3 above)
Comment (AdamPawlinski) [reply apps/preview/next/pages/showcases/Filters/Brand.tsx:16]: @dkacper Do you think it should be refactored in all filter blocks in the scope of this PR or in a separate task?

**Pull request merged**